dist/build_source_package.sh

```diff
@@ -92,6 +92,31 @@ build_srpm() {
     cp -v "${temp_dir}"/rpm/SRPMS/*.src.rpm "${output_dir}"/rpm/
 }
 
+build_pkgbuild() {
+    check_tools updpkgsums
+
+    mkdir -p "${temp_dir}"/pkg
+    sed \
+        -e "s/@VERSION@/${full_version}/g" \
+        -e "s/@TARBALL_NAME@/$(basename "${tarball}")/g" \
+        < pkg/PKGBUILD.in \
+        > "${temp_dir}"/pkg/PKGBUILD
+
+    sed \
+        -e "s/@BINDIR@/\/usr\/bin/g" \
+        -e "s/@SYSCONFDIR@/\/etc/g" \
+        < ipmi-fan-control.service \
+        > "${temp_dir}"/pkg/ipmi-fan-control.service
```

Comment:
This needed to happen before the PKGBUILD is run so that the file hashes are correct